Assets/Scripts/AudioManager.cs

```diff
@@ -0,0 +1,88 @@
+﻿using UnityEngine.Audio; 
+using UnityEngine;
+using System;
+
+public class AudioManager : MonoBehaviour
+{
+    public static AudioManager instance { get; private set; }
+    public Sound[] musicTracks;
+    public Sound[] sfxTracks;
+    
+
+    private void Awake()
+    {
+        if (instance == null)
+        {
+            instance = this;
+        }
+        else
+        {
+            Destroy(gameObject);
+            return; 
+        }
```

Comment:
AudioManager will need to be carried over between Scenes. I suggest we replace lines 14-22 with...
`if (instance != null && instance != this) 
{
    Destroy(gameObject);
    return;
}
else 
{
    instance = this;
}
DontDestroyOnLoad(gameObject);`

Reply:
Done